app/models/community.rb

```diff
@@ -568,8 +569,8 @@ def payments_in_use?
   end
 
   def default_currency
-    if available_currencies
-      available_currencies.gsub(" ","").split(",").first
+    if currency
```

Comment:
When you are done with the migrations, I'd set the currency to NOT NULL in database and at the same time remove this `if` block